compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/window/DesktopPopup.desktop.kt

```diff
@@ -28,8 +27,16 @@ import androidx.compose.ui.unit.IntRect
 import androidx.compose.ui.unit.IntSize
 import androidx.compose.ui.unit.LayoutDirection
 import androidx.compose.ui.unit.dp
-import java.awt.MouseInfo
-import javax.swing.SwingUtilities.convertPointFromScreen
+import androidx.compose.ui.unit.round
+
+/**
+ * Returns a remembered value of the mouse cursor position.
+ */
+@Composable
+private fun rememberCursorPosition(): IntOffset {
```

Comment:
Do we need `rememberCursorPositionProvider` API after the new API is stabilized? In my opinion, it is more robust and understandable, and we can deprecate it after the stabilization.

Maybe, we even don't have to remove AWT dependency out of it, at leave it as it is, because it isn't used in Compose now.

Reply:
It doesn't seem like `rememberCursorPositionProvider` will be needed when we remove the APIs that use it,

Reply:
Actually, we may need it after all. The Android version of `DropdownMenu` has this API:
```
fun DropdownMenu(
    expanded: Boolean,
    onDismissRequest: () -> Unit,
    modifier: Modifier = Modifier,
    offset: DpOffset = DpOffset(0.dp, 0.dp),
    properties: PopupProperties = PopupProperties(focusable = true),
    content: @Composable ColumnScope.() -> Unit
) 
```
and we need to keep a version that is consistent with that. In fact, now I'm not even sure we can provide an alternate API in the `material` module, can we?